.travis.yml

```diff
@@ -10,12 +10,6 @@ os:
 git:
   depth: 300
 
-before_script:
-  - npm run bootstrap
-
-script:
-  - npm test
```

Comment:
`npm test` is the default build script for Node.js projects, no need to explicitly specify it in our Travis config.